examples/ignition/k8s-master.yaml

```diff
@@ -363,13 +356,12 @@ storage:
                       },
                       {
                         "args": [
-                          "-machines=http://127.0.0.1:4001",
-                          "-addr=0.0.0.0:53",
-                          "-ns-rotate=false",
-                          "-domain=cluster.local."
+                          "--cache-size=1000",
+                          "--no-resolv",
+                          "--server=127.0.0.1#10053"
```

Comment:
Yeah, looks like I grabbed it from here too: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/skydns-rc.yaml.in#L91